azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1531,6 +1538,8 @@ public JobRunnerEventListener() {
           props.getString("jetty.hostname", "localhost")));
       metaData.put("jobProxyUser",
           jobRunner.getProps().getString(JobProperties.USER_TO_PROXY, null));
+      // attempt id
+      metaData.put("attempt ID", String.valueOf(node.getAttempt()));
```

Comment:
Currently a job execution attempt is a regular integer in the ExecutableNode class not a separate entity. Wondering if "attemptId" is a good name for it in that case. Also if there were multiple execution attempts will `getAttempt()` return the last of them?

Reply:
Yes, I am renaming it to attemptId.
Yes, the getAttempt() should give the last one. The attempt in ExecutableNode is Atomic Integer which must contain the latest value.

Reply:
What I wanted to convey is that attempt is a just number not an Id. Why do you prefer naming it "attemptId"? 
BTW, this number starts in zero, should we report it with a +1? Since attempt 0 might not be make much sense.
How did you verify that `getAttempt()` returns the last attempt?

Reply:
I replied to you on slack which should answer the last question. Please take a look at lowRunner.retryJobIfPossible() , ExecutableNode.resetForRetry() as well ExecutionAttempt class.
I don't understand what you mean by attempt is just a number not an id? This number is specific to this particular job and identifies which attempt it is for that job, hence attemptId.
Adding +1 to it may confuse the users as we show attempt 0 for job logs in UI. Also it is pretty common to start the count with 0.